src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -32,7 +32,7 @@ private void InitSort(CultureInfo culture)
             }
             else
             {
-                _isAsciiEqualityOrdinal = (_sortName == "en-US" || _sortName == "");
+                _isAsciiEqualityOrdinal = (_sortName == "en-US" || _sortName == "" || _sortName == "en-GB");
```

Comment:
Can we do this for all cultures that start with `en-` ? Single out US and GB here is non-inclusive...

Reply:
Good point, I've enabled it for all `en*` cultures (there is a culture called "en").